lambda/src/validateTx.js

```diff
@@ -48,6 +48,11 @@ async function checkBtcTxConfirmation (txHash) {
   }
 }
 
+async function checkLibraTxConfirmation (txHash) {
+  // libra tx is almost instant
+  return 1
```

Comment:
This could be problematic in the future. Although Libra is almost instant, this function should still try to obtain some sort of confirmation. We can set the time delay between each confirmation attempt to a minimum, like 1 second